content/docs/addons-test-utils.md

```diff
@@ -262,7 +262,7 @@ findRenderedDOMComponentWithTag(
 )
 ```
 
-Like [`scryRenderedDOMComponentsWithTag()`](#scryrendereddomcomponentswithtag) but expects there to be one result, and returns that one result, or throws exception if there is any other number of matches besides one.
+जैसे [`scryRenderedDOMComponentsWithTag()`](#scryrendereddomcomponentswithtag), लेकिन वहाँ एक परिणाम होने की उम्मीद करता है, और वह एक परिणाम देता है, या एक के अलावा किसी अन्य संख्या के मैच होने पर एक्सेप्शन थ्रो करता है।
```

Comment:
same as before

Reply:
fixed in f6949489e064deef0435f5b70bccff1711585cf2